web/template/create_draft.gohtml

```diff
@@ -49,21 +49,21 @@
           <input type="hidden" name="xsrfToken" value="{{ .XSRFToken }}"/>
 
           <div class="govuk-form-group {{ if .Error.Field.types }}govuk-form-group--error{{ end }}" id="f-types">
-            <fieldset class="govuk-fieldset" aria-describedby="f-type-hint">
+            <fieldset class="govuk-fieldset">
```

Comment:
Sorry, the ticket was unclear: `aria-describedby` on <fieldset> is fine, it's only the ones on <div> that we need to correct.